src/functionalTest/java/uk/gov/hmcts/probate/functional/tests/InvitationControllerFunctionalTests.java

```diff
@@ -54,7 +54,7 @@
     public TestRetryRule retryRule = new TestRetryRule(3);
 
     @Before
-    public void aInit() throws IOException, JSONException {
+    public void init() throws IOException, JSONException {
```

Comment:
whats the difference between aInit() or init()

Reply:
Regex wouldn't allow a single lower case at the start, must be 2 or more